### PR TITLE
Adapt settings parser and workaround for SAGA

### PR DESCRIPTION
**Pull request by emiliano-gelati**

### CODE REVIEW

**Comment by emiliano-gelati** [comment]
Related issues: #11 #12

Run:
- install.sh to install
- LandsitesTools/run_example.sh to test

File: LandsitesTools/interface_settings.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import re
+from configparser import ConfigParser, ExtendedInterpolation
+from pathlib import Path
+import pandas as pd
+from dateutil.parser import parse as parsetime
+
+
+def splitSequence(sequence, sep_regex='[ ,;]+'):
```
Comment (lasseke) [inline LandsitesTools/interface_settings.py:8]: Small question: the PEP-8 guidelines recommend using lower-case letters and underscores for function names: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Of course, camel-case is also completely fine, IMO we should just agree on one common scheme :-)

**Review by lasseke — COMMENTED** [review]
Looks great! Added one comment about naming conventions.

File: LandsitesTools/interface_settings.py (hunk 1 above)
Comment (emiliano-gelati) [reply LandsitesTools/interface_settings.py:8]: I think it is a good idea to follow PEP-8 conventions as @lasseke suggests. Let us decide tomorrow together, as anyway other changes may be needed to make the settings interface work properly with interactive modifications.

**Pull request merged**